ansible/roles/dashmate/templates/dashmate.json.j2

```diff
@@ -0,0 +1,238 @@
+{
+  "configFormatVersion": "0.24.0-dev.11",
```

Comment:
Can we make this version, and the dashmate version, variable in the config file? Please add a good example value to `lib/configGenerator/generateAnsibleConfig.js` as well to remind users to set this when setting the network parameters